paimon-core/src/test/java/org/apache/paimon/table/system/FilesTableTest.java

```diff
@@ -211,7 +211,8 @@ private List<InternalRow> getExpectedResult(long snapshotId) {
                                             maxCol1, maxKey, partition1, partition2)),
                             file.minSequenceNumber(),
                             file.maxSequenceNumber(),
-                            file.creationTime()));
+                            file.creationTime(),
+                            BinaryString.fromString(file.fileSource().get().toString())));
```

Comment:
Why not : 
BinaryString.fromString(file.fileSource().map(FileSource::toString).orElse(null))
?

Reply:
fixed